signer/src/transaction_coordinator.rs

```diff
@@ -365,30 +368,38 @@ where
 
     /// Construct and coordinate WSTS signing rounds for sBTC transactions on Bitcoin,
     /// fulfilling pending deposit and withdraw requests.
-    #[tracing::instrument(skip_all, fields(bitcoin_chain_tip))]
+    #[tracing::instrument(skip_all)]
     async fn construct_and_sign_bitcoin_sbtc_transactions(
         &mut self,
         bitcoin_chain_tip: &model::BitcoinBlockHash,
         aggregate_key: &PublicKey,
         signer_public_keys: &BTreeSet<PublicKey>,
     ) -> Result<(), Error> {
-        tracing::debug!("Fetching the stacks chain tip");
+        tracing::debug!("fetching the stacks chain tip");
         let stacks_chain_tip = self
             .context
             .get_storage()
             .get_stacks_chain_tip(bitcoin_chain_tip)
             .await?
             .ok_or(Error::NoStacksChainTip)?;
 
+        tracing::debug!(stacks_chain_tip = %stacks_chain_tip.block_hash, "retrieved the stacks chain tip");
```

Comment:
suggestion: i'd newline this

Reply:
https://github.com/stacks-network/sbtc/pull/874/commits/bb02a92a63ab68af348161c9cc504c7788e0654e